z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -279,7 +279,7 @@ Others are **optional**, you can enable them and update value if you need more s
 | `os_bootstrap_ip` | \<bootstrap ip addr\> |'x.x.x.x, <br>**required** when `auto_allocated_ip` is false
 | `os_master_ip` | \<master ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x], <br>**required** when `auto_allocated_ip` is false
 | `os_infra_ip` |\<infra ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x], <br>**required** when `auto_allocated_ip` is false
-| `use_proxy` |false|(Boolean) true or false, if true then Openshft Container Platform will use the proxy setting
+| `use_proxy` |false|(Boolean) true or false, if true then Openshft Container Platform will use the proxy setting, get detail from [doc.openshift.com](https://docs.openshift.com/container-platform/4.9/operators/admin/olm-configuring-proxy-support.html)
```

Comment:
this is talking about OLM proxy, I think we want to talk about cluster level proxy instead of OLM only?

Reply:
how about this url?
https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-configure-proxy_installing-bare-metal

Reply:
works for me